rmarkdown.Rmd

```diff
@@ -167,10 +282,18 @@ like: [@Hoejsgaard2006a]... the reference is inserted onto the bottom of the
 document. You need to add a line like this `bibliography: bibfile.bib` to the top
 YAML header (e.g. right below where you put the `title` and `author`)
 
-## Other resources
+#### Exercise 4: quick R calculations and inserting citations to your document
+Time: Until end of session
+
+Now you are asked:
+
+1. To calculate the correlation between Sepal.Length and Sepal.Width inline (outside of the R chunck). 
+2. To create a new document (in the same directory as this document is placed) and include a reference of the paper you most like! 
+3. Cite this paper in the **discussion section** of your document.
+4. Compile it again, and see what happens!
```

Comment:
Maybe have an additional task to change the theme of the document if they are knitting to html.

Reply:
I thought about that to, in order to not get in troubles with pandoc, then I suggest that they compile it to word document!